src/tools/rbd_mirror/PoolReplayer.cc

```diff
@@ -464,6 +469,13 @@ void PoolReplayer::print_status(Formatter *f, stringstream *ss)
     f->close_section();
   }
 
+  f->dump_string("local_ctx_admin_socket",
```

Comment:
@dillaman I am fine with dropping "ctx_" portion, as I don't like it very much too.Though don't you think it would be a little confusing for users, that is "local_admin_socket" vs "remote_admin_socket" would suggest the socket is located on local vs remote host?